error[E0599]: the method `print_message` exists for struct `ConstraintChain<"user", "___", policy::Enabled, dacquiri::prelude::EntityProof<"user", policy::User, ConstraintStore>>`, but its trait bounds were not satisfied
  --> tests/ui/fail/simple.rs:18:10
   |
18 |         .print_message();
   |          ^^^^^^^^^^^^^ method cannot be called on `ConstraintChain<"user", "___", policy::Enabled, dacquiri::prelude::EntityProof<"user", policy::User, ConstraintStore>>` due to unsatisfied trait bounds
   |
  ::: src/chain.rs
   |
   | pub struct ConstraintChain<const STAG: EntityTag, const RTAG: EntityTag, Attr, Next> {
   | ------------------------------------------------------------------------------------
   | |
   | doesn't satisfy `_: VerifiedUserPolicyMarker<_>`
   | doesn't satisfy `_: policy::VerifiedUserPolicy<_>`
   |
note: trait bound `ConstraintChain<"user", "___", policy::Enabled, dacquiri::prelude::EntityProof<"user", policy::User, ConstraintStore>>: VerifiedUserPolicyMarker<_>` was not satisfied
  --> tests/ui/fail/simple.rs:55:5
   |
55 |       #[policy(
   |  _____^
   | |_____|
   | |
56 | |         entities = (
57 | |             user: User
58 | |         ),
...  |
62 | |         )
63 | |     )]
   | |______^
   | |______|
   |
   = note: this error originates in the attribute macro `policy` (in Nightly builds, run with -Z macro-backtrace for more info)
